README.md

```diff
@@ -2527,6 +2533,9 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [Pro
 IEEE Robotics and Automation Letters (RA-L), 2021. [Dataset](http://rpg.ifi.uzh.ch/dsec.html), [PDF](http://rpg.ifi.uzh.ch/docs/RAL21_DSEC.pdf), [Code](https://github.com/uzh-rpg/DSEC), [Video](https://youtu.be/W4yW78y4F7A).
 - [Gao et al., RAL 2022](#Gao22ral), *VECtor: A Versatile Event-Centric Benchmark for Multi-Sensor SLAM*.
 
+### Monocular Voxel Reconstruction
```

Comment:
Similar changes would apply here.

Reply:
[620ac5c](https://github.com/uzh-rpg/event-based_vision_resources/pull/235/commits/620ac5ca12c38ed4e9327d86f183f907716c9aef)
Removed the comma between ICVR and 2023 in the “Monocular Voxel Reconstruction” subsection under the "Datasets and Simulators” section.